lib/Doctrine/ORM/Internal/CommitOrderCalculator.php

```diff
@@ -17,33 +20,28 @@
  */
 class CommitOrderCalculator
 {
-    public const NOT_VISITED = 0;
-    public const IN_PROGRESS = 1;
-    public const VISITED     = 2;
+    /** @deprecated */
+    public const NOT_VISITED = VertexState::NOT_VISITED;
+
+    /** @deprecated */
+    public const IN_PROGRESS = VertexState::IN_PROGRESS;
+
+    /** @deprecated */
+    public const VISITED = VertexState::VISITED;
```

Comment:
These should be mentioned in `UPGRADE.md`

Reply:
I wasn't sure if I even need a deprecation layer for them since this class is located in a namespace that literally says `Internal`.

Reply:
I've added a note to UPGRADE.md.

Reply:
Ah sorry, I didn't spot this. Wasn't required indeed.